### DISCUSSION TITLE
Welcome to F90 Simple Shop System Discussions!

### DISCUSSION THREAD

**Discussion by NikolaiF90** [NikolaiF90/F90SimpleShopSystem#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Valera8994** [comment]
how to make sure that when you take an object from the floor, money is awarded, for example, a pack of dollars?

**Reply by NikolaiF90** [reply]
Theres multiple of ways actually. But here's the basic idea. This one is done by scripting. 
Firat create an object. If me, I'll try createVehicle command. then in the same script addAction the object. 
 ```sqf
private _moneyRoll = createVehicle ["money_roll", position player, [], 0, "NONE"];
_moneyRoll addAction 
[
    "Pick up",
    {
        params ["_target", "_caller", "_actionId", "_arguments"];
        // Delete the object. Assume player had already taken it
        deleteVehicle _target;
        // Add money script
        // Maybe something like private _money = _caller getVariable "money";
    }
];
```
There's a whole lot needed for it to work as expected without bugs and so on. But this just the basic idea. But feel free to email me NikolaiPrinceF90@gmail.com maybe we can work together
